ops/ansible/playbooks-ccs/files/client_data_server_ab2d_imp_certificate.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN CERTIFICATE-----
+MIIEsDCCApgCCQDVY5whus7LJzANBgkqhkiG9w0BAQsFADAaMRgwFgYDVQQDDA9h
+YjJkLWltcC1jbGllbnQwHhcNMjAwMTIzMTQ0ODM4WhcNMjUwMTIxMTQ0ODM4WjAa
+MRgwFgYDVQQDDA9hYjJkLWltcC1jbGllbnQwggIiMA0GCSqGSIb3DQEBAQUAA4IC
+DwAwggIKAoICAQCcsXAZI4YfSK3soGNm79eg6wD723xj0+2xiSAqsFLN3lQoluK+
+0CVNE40lf6xKGSHLnEmzyvI4X2SB3+Am8CAKSXzkj4RQqD2WaxzyTccc5iXhhoIJ
+zVsIas5OoP4uce5s2/Y/AH2l04yC6pztEyL/DpUexhkxNcIj56sFqCrgyQglW116
+blZ7W5ESh3MwGF9oh5pxjU5Idd4KJSs9ekBXzAJzKro/S47yWDHn39oVvTNYkjqS
+c5VCW1ydfR/qJxDMxe8GmINrT/EIRil9DeUEk2a73XUpZl4aZHaf4MfecnQRFztO
+tJLW+I8YHkZz4aHukmkfEBTALIrNEnqGCCkd/3nwR5UHpRKvQ/VlqRmXrbdDqxbh
+URYNkOqu0mJvbRYc71Wpy+LgBKZXSv3CoY4npMNYP6+eh4YoHpKRYzwoT63Z+vfM
+ulRJArB9KcuTBrq/oX7HTKe49ukJ+w0Qa6+gREtPiGH8gRcCdozTuDOFFBBnxPIi
+5zU8Qu3nd5LFlppr5AY2r4i9YW6frmdKiKlauMdCUVBiifTHLy4Fz9yME6smkWmy
+5WVxrIzVcSwkQ2eCYQv9f9ubZS7OE5GZyEsPSiuFVV3ozQmB4sfvO4Ragd/+7VEU
+tZ+c/6W/fX/qPevShGnNSubiMCrwAnwzZ+PJoX8y4+kmG13g127QG6cL9QIDAQAB
+MA0GCSqGSIb3DQEBCwUAA4ICAQBWKEgTvfyJ6H2xq1DhUMFp/DTrl3IknlSweGVu
+yglC/USAHqeL2oYe1IJ3J19NuBXAsRAE8PhVm/fSm4L/l4Jdw6LXb9ODj9cZomzt
+b2JEXKEF9xHHd4ka2NWClcODdXK9S0oA4vrN9rzfr293WdYEHKhnTj7FBz4T4GqY
+t0fHDeiGu7pAI6+H2nJuUZmX20EXYns6zUR0Z4hMWKB8HTtqYE3hu42zOFpajtG9
+aAnspwnb3mv66QN1gFwPDpF7yhzA8yZKTud+gWCKvHUDJcmSjgQ16cRe/8IP6h0L
+xQn620jeSJsstkIMKtF/Cyh2yZagou3SeKMty2VklRXqRuNh7ufIVdyc+HbALYGb
+MbwVv2DsdykMD3zkOaWDrPQ2sLmFknRSAuLlZS1iuBoAtMCTTxFEkmRK61gGcJJi
+8hmDTIbxtA1jDwpic2lB2yRjYnddesYPuOsEiLhKoErBb4aoj7ebcAe6yNajNM/3
+qBM/TV4zxvlP4VW52XhQTEOY8sR/KZe4KyCfsQLSkRZaRruCv3egDpXjCMbodUrz
+YAKHhN+39X3tlBIyfQIn6t02zOksrAzIZZ23iOTIqOLs0vzSrxsRD0pbmF+KjFcn
+xRsAXUHSKEyIBJt4aSIEBaR01X3pxveAq0PyM5uR156thHJmIsVZFdRvHbotbuQL
+m2Xpzw==
+-----END CERTIFICATE-----
```

Comment:
```
openssl x509 -text -in ops/ansible/playbooks-ccs/files/client_data_server_ab2d_imp_certificate.pem
Certificate:
    Data:
        Version: 1 (0x0)
        Serial Number: 15376305221454973735 (0xd5639c21bacecb27)
    Signature Algorithm: sha256WithRSAEncryption
        Issuer: CN=ab2d-imp-client
        Validity
            Not Before: Jan 23 14:48:38 2020 GMT
            Not After : Jan 21 14:48:38 2025 GMT
        Subject: CN=ab2d-imp-client
        Subject Public Key Info:
            Public Key Algorithm: rsaEncryption
                Public-Key: (4096 bit)
                Modulus:
                    00:9c:b1:70:19:23:86:1f:48:ad:ec:a0:63:66:ef:
                    d7:a0:eb:00:fb:db:7c:63:d3:ed:b1:89:20:2a:b0:
                    52:cd:de:54:28:96:e2:be:d0:25:4d:13:8d:25:7f:
                    ac:4a:19:21:cb:9c:49:b3:ca:f2:38:5f:64:81:df:
                    e0:26:f0:20:0a:49:7c:e4:8f:84:50:a8:3d:96:6b:
                    1c:f2:4d:c7:1c:e6:25:e1:86:82:09:cd:5b:08:6a:
                    ce:4e:a0:fe:2e:71:ee:6c:db:f6:3f:00:7d:a5:d3:
                    8c:82:ea:9c:ed:13:22:ff:0e:95:1e:c6:19:31:35:
                    c2:23:e7:ab:05:a8:2a:e0:c9:08:25:5b:5d:7a:6e:
                    56:7b:5b:91:12:87:73:30:18:5f:68:87:9a:71:8d:
                    4e:48:75:de:0a:25:2b:3d:7a:40:57:cc:02:73:2a:
                    ba:3f:4b:8e:f2:58:31:e7:df:da:15:bd:33:58:92:
                    3a:92:73:95:42:5b:5c:9d:7d:1f:ea:27:10:cc:c5:
                    ef:06:98:83:6b:4f:f1:08:46:29:7d:0d:e5:04:93:
                    66:bb:dd:75:29:66:5e:1a:64:76:9f:e0:c7:de:72:
                    74:11:17:3b:4e:b4:92:d6:f8:8f:18:1e:46:73:e1:
                    a1:ee:92:69:1f:10:14:c0:2c:8a:cd:12:7a:86:08:
                    29:1d:ff:79:f0:47:95:07:a5:12:af:43:f5:65:a9:
                    19:97:ad:b7:43:ab:16:e1:51:16:0d:90:ea:ae:d2:
                    62:6f:6d:16:1c:ef:55:a9:cb:e2:e0:04:a6:57:4a:
                    fd:c2:a1:8e:27:a4:c3:58:3f:af:9e:87:86:28:1e:
                    92:91:63:3c:28:4f:ad:d9:fa:f7:cc:ba:54:49:02:
                    b0:7d:29:cb:93:06:ba:bf:a1:7e:c7:4c:a7:b8:f6:
                    e9:09:fb:0d:10:6b:af:a0:44:4b:4f:88:61:fc:81:
                    17:02:76:8c:d3:b8:33:85:14:10:67:c4:f2:22:e7:
                    35:3c:42:ed:e7:77:92:c5:96:9a:6b:e4:06:36:af:
                    88:bd:61:6e:9f:ae:67:4a:88:a9:5a:b8:c7:42:51:
                    50:62:89:f4:c7:2f:2e:05:cf:dc:8c:13:ab:26:91:
                    69:b2:e5:65:71:ac:8c:d5:71:2c:24:43:67:82:61:
                    0b:fd:7f:db:9b:65:2e:ce:13:91:99:c8:4b:0f:4a:
                    2b:85:55:5d:e8:cd:09:81:e2:c7:ef:3b:84:5a:81:
                    df:fe:ed:51:14:b5:9f:9c:ff:a5:bf:7d:7f:ea:3d:
                    eb:d2:84:69:cd:4a:e6:e2:30:2a:f0:02:7c:33:67:
                    e3:c9:a1:7f:32:e3:e9:26:1b:5d:e0:d7:6e:d0:1b:
                    a7:0b:f5
                Exponent: 65537 (0x10001)
    Signature Algorithm: sha256WithRSAEncryption
```
There's no private key material here, right? ✅
Is the cert different from any other certs from the same project (just watching out for oopses)? ✅
Is its Subject unique? ✅
Does it have a reasonably useful/informative Subject? ✅
Does it have a reasonable key size (>= 2048)? ✅